backend/kernelCI_app/management/commands/helpers/kcidbng_ingester.py

```diff
@@ -268,7 +301,9 @@ def process_submission_item(data: dict[str, Any], metadata: dict[str, Any]):
         logger.info("Ingested %s in %.2f KB/s", metadata["filename"], ing_speed)
 
 
-def db_worker(stop_event: threading.Event):
+def db_worker(  # noqa: C901
+    stop_event: threading.Event,
+):
```

Comment:
nit, I'm almost sure you could've put the noqa in front.

```suggestion
def db_worker(stop_event: threading.Event):  # noqa: C901
```

But I'd argue that the c901 alert was accurate, and we should try to refactor this function in the future